site2/docs/io-quickstart.md

```diff
@@ -1,136 +1,96 @@
 ---
 id: io-quickstart
-title: "Tutorial: Connecting Pulsar with Apache Cassandra"
-sidebar_label: Getting started
+title: "Tutorial: Connect Pulsar with Database"
+sidebar_label: Get started
 ---
 
-This tutorial provides a hands-on look at how you can move data out of Pulsar without writing a single line of code.
-It is helpful to review the [concepts](io-overview.md) for Pulsar I/O in tandem with running the steps in this guide
-to gain a deeper understanding. At the end of this tutorial, you will be able to:
+This tutorial provides a hands-on look at how you can move data out of Pulsar without writing a single line of code.  
 
-- Connect your Pulsar cluster with your Cassandra cluster
+It is helpful to review the [concepts](io-overview.md) for Pulsar I/O with running the steps in this guide to gain a deeper understanding.   
+
+At the end of this tutorial, you will be able to:
+
+- [Connect Pulsar to Apache Cassandra](#Connect-Pulsar-to-Apache-Cassandra)
+  
+- [Connect Pulsar to MySQL](#Connect-Pulsar-to-MySQL)
 
 > #### Tip
 >
-> 1. These instructions assume you are running Pulsar in [standalone mode](getting-started-standalone.md). However all
+> * These instructions assume you are running Pulsar in [standalone mode](getting-started-standalone.md). However all
 > the commands used in this tutorial should be able to be used in a multi-nodes Pulsar cluster without any changes.
 >
-> 2. All the instructions are assumed to run at the root directory of a Pulsar binary distribution.
-
-## Installing Pulsar
-
-To get started running Pulsar, download a binary tarball release in one of the following ways:
+> * All the instructions are assumed to run at the root directory of a Pulsar binary distribution.
 
-* by clicking the link below and downloading the release from an Apache mirror:
+# Install Pulsar and builtin connector
 
-  * <a href="pulsar:binary_release_url" download>Pulsar {{pulsar:version}} binary release</a>
+Before connecting Pulsar to a database, we need to install Pulsar and the desired builtin connector.
 
-* from the Pulsar [downloads page](pulsar:download_page_url)
-* from the Pulsar [releases page](https://github.com/apache/pulsar/releases/latest)
-* using [wget](https://www.gnu.org/software/wget):
+* For more information about how to install a standalone Pulsar, see [here](https://pulsar.apache.org/docs/en/standalone/#installing-pulsar).
```

Comment:
have you build the website locally? I think there are 2 issues here.

a) "https" => I don't think apache supports `https` for project website. 
b) I would not recommend putting a direct http url here. you should reference the desired markdown file. in this line, it should be:

```
[here](getting-started-standalone.md#installing-pulsar)
```

(although I haven't verified if the website framework can handle this correctly or not, you'd better build the website locally to see what is the final result)

Reply:
Yes, I've built the website locally, `https` works.

But I've changed them to reference `.md` files because URL might be changed in the future.

Reply:
ok we shouldn't avoid referencing any absolute url. there are multiple reasons:

1) you will be redirected to a https link when browsing an insecure http web page.
2) pulsar is a multi-versioned documentation site. at the point you put down an absolute url, the link always point to current version. that cause confusion when a user browse the documentation of a given version. so you should also use relative links. hence you should use `[here](standalone/#installing-pulsar)`